types/luxon/index.d.mts

```diff
@@ -0,0 +1,7 @@
+// Minimum TypeScript Version: 4.3
+
+import * as luxon from "./index.js";
+
+export default luxon;
+
+export as namespace luxon;
```

Comment:
Shouldn't this just be
```suggestion
export * from "./index.js";
```
Luxon does not have a default export.

And the `export as namespace` is for globals, which is covered by the index.d.ts file...right?